Update readme to show the support status of the new esps (#9309)
adafruit/circuitpython: README.rst

Ports include the code unique to a microcontroller line.

The following ports are available: ``atmel-samd``, ``cxd56``, ``espressif``, ``litex``, ``mimxrt10xx``, ``nordic``, ``raspberrypi``, ``renode``, ``silabs`` (``efr32``), ``stm``, ``unix``.

> However, not all ports are not fully functional. Some have limited limited functionality and known serious bugs.

Reviewer: ```suggestion
However, not all ports are fully functional. Some have limited limited functionality and known serious bugs.
```

Ports include the code unique to a microcontroller line.

> The following ports are available: ``atmel-samd``, ``cxd56``, ``espressif``, ``litex``, ``mimxrt10xx``, ``nordic``, ``raspberrypi``, ``silabs`` (``efr32``), ``stm``, ``unix``.

Reviewer: ```suggestion
The following ports are available: ``atmel-samd``, ``cxd56``, ``espressif``, ``litex``, ``mimxrt10xx``, ``nordic``, ``raspberrypi``, ``renode``, ``silabs`` (``efr32``), ``stm``, ``unix``.
```

> However they are not all stable. For details, refer to the `latest release <https://github.com/adafruit/circuitpython/releases/latest>`__ notes.

Reviewer: ```suggestion
However, not all ports are not fully functional. Some have limited limited functionality and known serious bugs.
For details, refer to the **Port status** section in the `latest release <https://github.com/adafruit/circuitpython/releases/latest>`__ notes.
```
